src/librustc_mir/build/scope.rs

```diff
@@ -107,6 +107,9 @@ pub struct Scope<'tcx> {
     /// `ScopeAuxiliary`, but kept here for convenience
     extent: CodeExtent,
 
+    /// Whether there's anything for the cleanup path.
```

Comment:
> This field is true iff there's anything to be done for this scope when unwinding.

Can you make the comment say that, and also mention that we don't emit StorageDead when unwinding because clang doesn't do it?